### PR TITLE
Integrate age maf to vcf

### PR DESCRIPTION
**Pull request by zrm22**
## Description

Integrating in AGC archival sequence extraction.

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [x] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Updating create-maf-vcf to use the AGC archive for sequence extraction.
```

### CODE REVIEW

File: src/test/kotlin/net/maizegenetics/phgv2/cli/CreateMafVCFTest.kt [hunk 1]
```diff
@@ -118,14 +125,26 @@ class CreateMafVCFTest {
...
+
+
+        val agcCompress = AgcCompress()
+        // Create the initial compressed file
+        agcCompress.test("--fasta-list ${fastaCreateFileNamesFile} --db-path ${dbPath} --ref-fasta ${refFasta}")
+
+
         val createMAFVCF = CreateMafVcf()
-        createMAFVCF.test("--db-path data/test/buildMAFVCF/B97_ASM_Test.fa --bed data/test/buildMAFVCF/B73_Test.bed --reference data/test/buildMAFVCF/B73_Test.fa --maf-dir data/test/buildMAFVCF/mafs/ -o ${TestExtension.testVCFDir}")
+//        createMAFVCF.test("--db-path data/test/buildMAFVCF/B97_ASM_Test.fa --bed data/test/buildMAFVCF/B73_Test.bed --reference data/test/buildMAFVCF/B73_Test.fa --maf-dir data/test/buildMAFVCF/mafs/ -o ${TestExtension.testVCFDir}")
```
Comment (lynnjo) [inline src/test/kotlin/net/maizegenetics/phgv2/cli/CreateMafVCFTest.kt:140]: remove commented out line?

File: src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt [hunk 2]
```diff
@@ -131,10 +135,10 @@ class CreateMafVcf : CliktCommand(help = "Create gVCF and hVCF from Anchorwave M
         return gvcfVariantsByContig.keys
             .sorted() //Need to do a sort here as we need to make sure we process the chromosomes in order
```
Comment (lynnjo) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt:136]: Comment on using sort:  How will this sort chr1, chr9, chr10?  In Biokotlin we handled this with a alphaThenNumeric sort.  I'm using it in places like this:

var sortedRecords = records.sortedWith(compareBy(SeqRangeSort.alphaThenNumberSort){ name:String -> name})

Something to consider if we need to ensure chr9 is sorted before chr10.

**Review by lynnjo — APPROVED** [review]

File: src/test/kotlin/net/maizegenetics/phgv2/cli/CreateMafVCFTest.kt (hunk 1 above)
Comment (zrm22) [reply src/test/kotlin/net/maizegenetics/phgv2/cli/CreateMafVCFTest.kt]: Removed

File: src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt (hunk 2 above)
Comment (zrm22) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt]: Didn't know this was a thing, added it in here and something similar to when we are loading up the BED file to sort the chroms correctly.

**Review by tcasstevens — APPROVED** [review]

**Pull request merged**